cpp/installer.cc

```diff
@@ -958,67 +971,63 @@ void ConfigureFullTextBackend(const bool production, const bool install_cronjobs
 
 
 int Main(int argc, char **argv) {
-    if (argc < 3 or argc > 5)
+    if (argc < 2)
         Usage();
 
+    InstallationType installation_type(UB_TOOLS_ONLY);
     std::string vufind_system_type_string;
     VuFindSystemType vufind_system_type(IXTHEO);
     bool omit_cronjobs(false);
     bool omit_systemctl(false);
-
-    bool production;
-    if (std::strcmp("--production", argv[1]) == 0)
-        production = true;
-    else if (std::strcmp("--test", argv[1]) == 0)
-        production = false;
+    bool production(false);
```

Comment:
Variables should be declared as close to their initial use as possible in order to reduce their scope and to make it easy for the reader to see and remember their type.